.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+# We do not use Github for issues!
+
+Please do not create issues here.
+
+## To get help
+
+If you need help using [Archive of Our Own](http://archiveofourown.org), please [contact its Support team](http://archiveofourown.org/support)
+
+## To suggest a new feature
+
+[AO3 Support](http://archiveofourown.org/support) also track feature requests.
```

Comment:
I think this is an American/British English thing -- American English would use the singular verb "tracks" with a collective noun like "team." Wikipedia seems to confirm my suspicions https://en.wikipedia.org/wiki/Collective_noun